### PR TITLE
CPDP-1859: [POA] Indirect disposals content changes

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -2223,6 +2236,7 @@ acquisitionPriceBought.agent.indirect.title=What was the acquisition amount?
 acquisitionPriceBought.trust.indirect.title=What was the acquisition amount?
 acquisitionPriceBought.capacitor.indirect.title=How much did the person pay for the shares?
 acquisitionPriceBought.personalRep.indirect.title=How much did the person pay for the shares?
+acquisitionPriceBought.personalRep.indirect.title=
```
Comment (AndyHWChung) [inline conf/messages:2239]: is this supposed to be here?

Comment (AndyHWChung) [reply conf/messages:2239]: are we missing a `personalRepInPeriodOfAdmin` and `personalRepInPeriodOfAdmin.agent` titles here?

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsControllerSpec.scala [hunk 2]
```diff
@@ -140,6 +141,39 @@ class CompanyDetailsControllerSpec
...
+    val sessionData      = SessionData.empty.copy(
+      journeyStatus = Some(fillingOutReturn)
+    )
+
+    (sessionData, fillingOutReturn, draftReturn)
+  }
+
+  val acceptedIndividualUserTypeGen: Gen[IndividualUserType] =
+    individualUserTypeGen.filter {
+      case Self | Capacitor | PersonalRepresentative => true
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsControllerSpec.scala:173]: we should be able to update this to include `PeronslaRepresentativeInPeriodOfAdmin` now. But then that's all `IndividualUserType`, so we might not need `acceptedIndividualUserTypeGen` at all now

Comment (rambabu-posa) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsControllerSpec.scala:173]: As this spec is not organised  properly to handle different User types and IndividualUserTypes, I have tested Non-POA using acceptedIndividualUserTypeGen and PeronslaRepresentativeInPeriodOfAdmin in separate tests.

Will try to merge them into one.

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**